src/umpire/ResourceManager.cpp

```diff
@@ -115,9 +115,10 @@ ResourceManager::initialize()
       UMPIRE_VERSION_MINOR << "." <<
       UMPIRE_VERSION_PATCH);
 
-  UMPIRE_REPLAY( "Umpire v" << UMPIRE_VERSION_MAJOR 
-      << "." << UMPIRE_VERSION_MINOR
-      << "." << UMPIRE_VERSION_PATCH);
+  UMPIRE_REPLAY( "\"event\": \"version\", \"result\": { \"major\":" << UMPIRE_VERSION_MAJOR 
```

Comment:
Should this be "payload" rather than "result"?

Reply:
Yeah, I guess I could see that.  I'll change it.

Reply:
This is now done.  @davidbeckingsale, please review.